app/views/command-line.html

```diff
@@ -48,7 +48,7 @@ <h1 id="cli">Command Line Tools</h1>
                   <div ng-show="showSessionToken" class="alert alert-warning">
                     <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
                     <strong>A token is a form of a password.</strong>
-                    Do not share your API token.
+                    Do not share your API token. <a href="#" ng-click="toggleShowSessionToken()" ng-show="showSessionToken">Click to hide the token...</a>
```

Comment:
I'd probably just say "Hide token".

You don't need the `ng-show` here since it's on the parent `div`.

You only need `href=""`. (Do you mind fixing the reveal `href` above as well?)